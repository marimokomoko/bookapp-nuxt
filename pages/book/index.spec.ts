import { mount } from '@vue/test-utils'
import Index from '@/pages/book/index.vue'
describe('画面レイアウト', ()=>{
    it('スナップショットテスト', ()=>{
        const wrapper = mount(Index)
        expect(wrapper.html()).toMatchSnapshot()
    })
})