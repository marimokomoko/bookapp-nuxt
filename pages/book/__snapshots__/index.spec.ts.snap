// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `
"<div>
  <h1>BookIndex</h1>
  <v-row>
    <v-col cols=\\"8\\">
      <v-btn color=\\"primary\\" to=\\"/book/search\\" class=\\"mb-3\\">検索する</v-btn>
    </v-col>
    <v-col cols=\\"4\\">
      <v-btn color=\\"error\\"> 削除する </v-btn>
    </v-col>
  </v-row>
  <v-row></v-row>
</div>"
`;
